### ISSUE TITLE
Missing payment methods

### ISSUE THREAD

**Issue by tecufanujacu** [marketplacekit/marketplacekit#123]
I installed this because of the paypal integration but after installing I discovered that there isn't any payment method available, the payment providers list is totally empty.

What is happening? Why the payment providers have been removed?

**Comment by Ezyweb-uk** [comment]
I've not found Paypal, but I did find Stripe [here](https://github.com/marketplacekit/stripe). It seems to have installed ok after I changed the module directory name from 'stripe-master' to 'Stripe', but I've not tested it yet. I'd like to have Paypal too!

**Comment by tecufanujacu** [comment]
Yes, me too I was been able to install Stripe in the same way as you, but it doesn't seem to work, and anyway it isn't the same thing as PayPal.
About PayPal, it seems that it has been removed, reading old issues in the past it was present.

So at the moment there isn't any valid payment method, it is a shame because this seem to be a great piece of software but it is discontinued and only partially working.

**Comment by Ezyweb-uk** [comment]
@tecufanujacu we're obviously looking for a similar system, you're welcome to email me. I've got some info back from Marketplacekit support.

**Comment by tecufanujacu** [comment]
At the moment I'm using the version 1.0.1-beta in which almost everything seems to be working and it is a good base on which start coding.
